adapters/socketio_berkeley.c

```diff
@@ -650,7 +652,11 @@ int socketio_setoption(CONCRETE_IO_HANDLE socket_io, const char* optionName, con
         }
         else if (strcmp(optionName, "tcp_keepalive_time") == 0)
         {
+#ifdef __APPLE__
```

Comment:
Ok, no problem.

---
In reply to: [75620983](https://github.com/Azure/azure-c-shared-utility/pull/21#discussion_r75620983) [](ancestors = 75620983)